src/metabase/sync/field_values.clj

```diff
@@ -23,7 +23,11 @@
 
 (s/defn ^:private update-field-values-for-field! [field :- i/FieldInstance]
   (log/debug (u/format-color 'green "Looking into updating FieldValues for %s" (sync-util/name-for-logging field)))
-  (field-values/create-or-update-full-field-values! field))
+  (let [field-values (db/select-one FieldValues :field_id (u/the-id field) :type :full)]
+    (if (field-values/inactive? field-values)
```

Comment:
```suggestion
    (if (and field-values (field-values/inactive? field-values))
```
in case you change `inactive?` to handle time check only then we should have a nil check here.